spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkSchemaUtil.java

```diff
@@ -319,7 +319,12 @@ public static void validateMetadataColumnReferences(Schema tableSchema, Schema r
   }
 
   public static Map<Integer, String> indexQuotedNameById(Schema schema) {
+    return indexQuotedNameById(schema, null);
+  }
+
+  public static Map<Integer, String> indexQuotedNameById(Schema schema, Set<Integer> fieldIds) {
```

Comment:
Do we need the original method if we are passing `fieldIds` in all internal uses? It hasn't been in a release so we can just remove it.